pkg/deploy/config.go

```diff
@@ -74,6 +74,7 @@ type Configuration struct {
 	GlobalResourceGroupName            *string                `json:"globalResourceGroupName,omitempty" value:"required"`
 	GlobalResourceGroupLocation        *string                `json:"globalResourceGroupLocation,omitempty" value:"required"`
 	GlobalSubscriptionID               *string                `json:"globalSubscriptionId,omitempty" value:"required"`
+	GlobalDevopsManagedIdentity        *string                `json:"globalDevopsManagedIdentity,omitempty" value:"required"`
```

Comment:
I also left a comment in PR 10887667 for RP-Config with this question, but cross-posting it here for visibility.

This new config which is being added to RP-Config is already present in RP-Config as `ev2Configuration.deploymentSettings.managedIdentityName`. The new config is duplicating that existing value. 

If we change one config in the future, we will need to remember to change the duplicated one too. Is there a technical or design reason not to use the existing config to avoid that duplication?

Reply:
Hey, sorry for the delay. So actually, the field is indeed there in the rp-config however, it is in the ev2Configuration section which is used only for ev2 stuff and we don't even have it in our model/struct in our go code and that is the reason, I created another field in order to keep things segregated.